apps/condo/domains/ticket/components/TicketChanges/TicketChange.tsx

```diff
@@ -0,0 +1,178 @@
+    const fields = [
+        ['clientPhone', ClientPhoneMessage],
+        ['details', DetailsMessage],
+        ['clientName', ClientNameMessage],
+        ['isPaid', IsPaidMessage],
+        ['isEmergency', IsEmergencyMessage],
+        ['statusDisplayName', StatusDisplayNameMessage],
+        ['unitName', UnitNameMessage],
+        ['assigneeDisplayName', AssigneeMessage],
+        ['classifierDisplayName', ClassifierMessage],
+    ]
+
+    const BooleanToString = {
+        isPaid: {
+            'true': intl.formatMessage({ id: 'pages.condo.ticket.TicketChanges.isPaid.true' }),
+            'false': intl.formatMessage({ id: 'pages.condo.ticket.TicketChanges.isPaid.false' }),
+        },
+        isEmergency: {
+            'true': intl.formatMessage({ id: 'pages.condo.ticket.TicketChanges.isEmergency.true' }),
+            'false': intl.formatMessage({ id: 'pages.condo.ticket.TicketChanges.isEmergency.false' }),
+        },
+    }
+
+    const formatField = (field, value) => {
+        const formatterFor = {
+            clientPhone: (field, value) => (
+                <PhoneLink value={value}/>
+            ),
+            details: (field, value) => (
+                value.length > 30 ? value.slice(0, 30) + '…' : value
+            ),
+            unitName: (field, value) => (
+                // Formally, unit name (e.g. apartment) was changed, but semantically,
+                // was changed a whole address of the ticket, so, to preserve context,
+                // the change of the unit is displayed as the change of the address
+                UnitNameChangedMessage
+                    .replace('{address}', ticketChange.ticket.property.address)
+                    .replace('{unitName}', value)
+            ),
+        }
+        return has(formatterFor, field)
+            ? formatterFor[field](field, value)
+            : value
+    }
+
+    const format = (field, value) => (
+        typeof value === 'boolean'
+            ? BooleanToString[field][value]
+            : formatField(field, value)
+    )
+
+    /*
+        Interpolates message string with JSX tags.
+        They will be safely mounted in place of `{to}` and `{from}` placeholders
+     */
+    const formatDiffMessage = (field, message, ticketChange) => {
+        // we have both "from" and "to" parts to interpolate
+        if (message.search('{from}') !== -1 && message.search('{to}') !== -1) {
+            const parts1 = message.split('{from}')
+            const parts2 =  parts1[1].split('{to}')
+            const valueFrom = ticketChange[`${field}From`]
+            const valueTo = ticketChange[`${field}To`]
+            return [
+                <SafeUserMention createdBy={ticketChange.createdBy} key={1}/>,
+                ' ',
+                parts1[0],
+                <del key={3}>{format(field, valueFrom)}</del>,
+                parts2[0],
+                <ins key={2}>{format(field, valueTo)}</ins>,
+                parts2[1],
+            ]
```

Comment:
```suggestion
           return (
               <>
                   <SafeUserMention createdBy={ticketChange.createdBy}/>
                   &nbsp;{parts1[0]}
                   <del >{format(field, valueFrom)}</del>
                   {parts2[0]}
                   <ins>{format(field, valueTo)}</ins>
                  {parts2[1]}
              </>
    )
```

Reply:
use ant Typography.Text and styles from theese components fro ```del``` and ```ins```.
https://ant.design/components/typography/

Reply:
I've already looked at this page, — there is no component for `<ins>`.
For what use case you don't want to use pure HTML-tags for elementary components?

Reply:
Changed to fragment in `f0ec5c70`

Reply:
<Text underline>Ant Design (underline)</Text>